operations/mimir-mixin/dashboards/dashboard-utils.libsonnet

```
@@ -238,12 +241,38 @@ local utils = import 'mixin-utils/utils.libsonnet';
       seriesOverrides: [
         {
           alias: 'request',
-          color: '#FFC000',
+          color: resourceRequestColor,
+          fill: 0,
+        },
+        {
+          alias: 'limit',
+          color: resourceLimitColor,
+          fill: 0,
+        },
+      ],
+      yaxes: $.yaxes('bytes'),
+      tooltip: { sort: 2 },  // Sort descending.
+    },
+
+  containerMemoryRSSPanel(title, containerName)::
+    $.panel(title) +
+    $.queryPanel([
+      // We use "max" instead of "sum" otherwise during a rolling update of a statefulset we will end up
```

Wouldn't this also apply to non-statefulset workloads (even if right now we happen to only use it for statefulsets).
```suggestion
      // We use "max" instead of "sum" otherwise during a rolling update of a workload we will end up
```

---

Deployments are not affected because the pod ID changes when rolling out. StatefulSets preserves the pod ID.